db/migrate/20160726034132_add_last_logged_in_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastLoggedInToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :last_logged_in, :datetime
```

Comment:
I am not using this feature right now, but I plan to sort users by their last activity on LeagueSpec and it will be used then. It will also be used to apply the Yellow Activity Icon if they have been recently active but not currently online